pxc/jenkins/pxc-package-testing-parallel.groovy

```diff
@@ -62,29 +60,43 @@ pipeline {
             description: "Node in which to test the product"
         )
 
+        choice(
```

Comment:
We have agreed that there won't be test_repo param in parallel job. We can just leave experimental in case we need to use it instead of testing.

Reply:
Discussed and left